src/server/middlewares/version.js

```diff
@@ -23,6 +23,7 @@ module.exports = async (req, res, next) => {
 
     try {
       doc = await AsyncAPIParser.parse(req.body, {
+        path: req.header('AsyncAPI.baseUrl'),
```

Comment:
Minor change: could you please use a name like `x-asyncapi-base-url` for the header? Just to follow conventions on custom headers.

Reply:
renamed

Reply:
Looking great now but your code linter messed things up and changed a bunch of things. Can you please fix it back?